File: fc-chat/demo/part3/ch3/05-1-message-system-add-channel/src/main/java/net/prostars/messagesystem/dto/websocket/outbound/QuitResponse.java
package net.prostars.messagesystem.dto.websocket.outbound;

import net.prostars.messagesystem.constant.MessageType;
import net.prostars.messagesystem.dto.domain.ChannelId;

public class QuitResponse extends BaseMessage {

  private final ChannelId channelId;

  public QuitResponse(ChannelId channelId) {
    super(MessageType.QUIT_RESPONSE);
    this.channelId = channelId;
  }

  public ChannelId getChannelId() {
    return channelId;
  }
}

File: fc-chat/demo/part3/with-push/src/main/kotlin/com/yscorp/withpush/messagesystem/dto/websocket/outbound/QuitResponse.kt
package com.yscorp.withpush.messagesystem.dto.websocket.outbound

import com.yscorp.withpush.messagesystem.constant.MessageType
import com.yscorp.withpush.messagesystem.dto.domain.ChannelId

class QuitResponse(val channelId: ChannelId) : BaseMessage(MessageType.QUIT_RESPONSE) {
}
